File: ExamenHeartstone_Kotlin/src/main/kotlin/models/personajes/Personaje.kt
package models.personajes

import models.Item
import java.time.LocalDateTime
import java.util.*

open class Personaje(
    val id: UUID,
    val fechaCreacion: LocalDateTime,
    val nombre: String,
    var vida: Int,
    val nivel: Int,
    val listaItems: MutableList<Item>
){
    open fun mostrarEstado(): String {
        return "Personaje(id=$id, fechaCreacion=$fechaCreacion, nombre='$nombre', vida=$vida, nivel=$nivel y listaItems=${listaItems.sortedBy { it.fechaCreacion }})"
    }
}

File: ExamenMoria-Python/projecto/models/personajes/Personaje.py
class Personaje:
    def __init__(self, nombre: str, estado: bool, listaItems: list):
        self.nombre = nombre
        self.estado = estado
        self.listaItems = listaItems

    def __str__(self) -> str:
        return f"Persona({self.nombre}, {self.estado}, {self.listaItems[0]})"